### ISSUE TITLE
voice recording started sound is not on by default

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#207639]
This should be `on` by default

![Image](https://github.com/microsoft/vscode/assets/29464607/4cbd9f7b-fcfe-4622-8c56-6d6d7b4b8749)
